cogs/conversioncog.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext import commands
+from unit_converter.converter import convert, converts
+
+
+class ConversionCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot  # This line in the constructor is required
+
+    @commands.command()
+    async def convert(
+        self, ctx, inputunit, outputunit
+    ):  # the ctx parameter is short for 'context'
+        """type your input and output units"""
+        save = convert(inputunit, outputunit)

```

Comment:
Please try and use variable names that accurately describe the data to which they refer. This benefits you and others who might try and read, edit, and/or improve your code in the future :slightly_smiling_face: